src/logical-assignment-private/nullish.case

```diff
@@ -0,0 +1,34 @@
+// Copyright 2022 Igalia S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+templates:
```

Comment:
This seems like it's a fairly brittle way to express what I wanted to achieve, which is "use all the templates in `default/` except for `method.template`. I didn't see a better way to do this, but let me know if there is.

Reply:
Did you try using a glob expression? `template: expr`?

Reply:
I did look at that, but crafting a glob expression that signified "`default/*` except for one file in there" seemed worse.